api/services/Reddit.js

```diff
@@ -123,6 +129,211 @@ exports.setFlair = function (refreshToken, name, cssClass, text, sub, callback)
     callback("Error retrieving token.");
   });
 };
+exports.banUser = function (refreshToken, username, ban_message, note, subreddit, duration, callback) {
+  exports.refreshToken(refreshToken, function (token) {
+    var data = {
+      api_type: 'json',
+      ban_message: ban_message,
+      name: username,
+      note: note,
+      type: 'banned'
+    };
+    if (duration) {
+      data.duration = duration;
+    }
+    if (left < 100 && moment().before(resetTime)) {
```

Comment:
The number of calls isn't constant, unfortunately -- it's 11 + (however many TSV threads the user had). We could set the cutoff to 25, I suppose.